arrows/core/example_detector.cxx

```diff
@@ -131,7 +131,7 @@ detect( vital::image_container_sptr image_data) const
 {
   auto detected_set = std::make_shared< kwiver::vital::detected_object_set>();
 
-  const double ct = (double)d->m_frame_ct;
+  const double ct { static_cast<double>(d->m_frame_ct) };
```

Comment:
Why using brace init here and not assignment? That is a fairly unusual style...

Reply:
Done